### ISSUE TITLE
Auto-scaling within a region

### ISSUE THREAD

**Issue by nkev** [benbjohnson/litestream-read-replica-demo#1]
This is very cool!

I was just pondering into the future about how this would work with fly.io auto-scaling in both a single region as well as multiple regions. 

For example, imagine this sample app in the `ord` region set to horizontally auto-scale 1-10 nodes (1 master and up to 9 read replicas) but also replicate to the `lhr` and `syd` regions, which in turn auto-scale their reads as necessary. 

THAT would be a game changer.. :) I bet Kurt from fly.io is all over this idea..

**Comment by benbjohnson** [comment]
Thanks, @nkev! Kurt and I chat regularly about some fun ideas for deploying this stuff out. I'm working on integrating at a lower level so that the replication is managed transparently and the end user doesn't have to worry about integration. It should automagically work with any auto-scaling scheme too so that'll be fun. :)

**Comment by nkev** [comment]
Hey Ben,

Just wanted to let you know this demo seems to be down in 2 regions, preventing the `increment` button from working:

<img width="452" alt="image" src="https://user-images.githubusercontent.com/5410916/167765451-479792cb-be9c-44b9-9655-7cfbc40173a6.png">

The [fly.io status page](https://status.flyio.net/) says all is well:
 
<img width="1021" alt="image" src="https://user-images.githubusercontent.com/5410916/167765566-1505f3f8-70a5-4d03-b26c-650bb25b5520.png">

I know it's only a quick demo you knocked up, but this is the kind of scenario the app should handle gracefully.

**Comment by benbjohnson** [comment]
@nkev Yes, you're correct. There were some bugs that came out in the beta so I'm refactoring the read replication a bit. I ended up taking down the demo for now until I get those issues resolved.
